internal/infrastructure/db/migrations/20240502181547_add_slug_to_device_definitions.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose StatementBegin
+
+SELECT 'up SQL query';
+SET search_path = device_definitions_api, public;
+ALTER TABLE device_definitions ADD name_slug VARCHAR(100) null;
+
+UPDATE device_definitions AS dd
+SET name_slug = dm.name_slug || '-' || dd.model_slug || '-' || CAST(dd.year AS VARCHAR(4))
+    FROM device_makes AS dm
+WHERE dd.device_make_id = dm.id;
```

Comment:
after the value is set, could we make the property NOT NULL ? I forget if this will need to be a separate migration.

Reply:
I added a new sentence to script migration